go-apps/meep-webhook/webhook.go

```diff
@@ -103,13 +105,9 @@ func loadConfig(configFile string) (*Config, error) {
 	return &cfg, nil
 }
 
-// Retrieve App Name from provided network element name string, if any
-func getAppName(name string) string {
-	names := bytes.Split([]byte(name), []byte("meep-"+activeScenarioName+"-"))
-	if len(names) != 2 {
-		return ""
-	}
-	return string(names[1])
+// Determine if resource is part of the active scenario
+func isScenarioResource(name string, namespace string) bool {
+	return name != "" && strings.HasPrefix(name, "meep-"+SandboxName+"-"+activeScenarioName+"-")
```

Comment:
Webhook works as follows:
- Intercepts & mutates Deployments & StatefulSets @ creation time
- Retrieves "release" label from k8s resource
  - Release label == helm chart release name
  - Helm chart release name created by Virt Engine == "meep-\<sandbox name\>-\<scenario name\>"
  - NOTE: This requires user charts to set "release" label to {{ .Release.Name }}
- If "release" label matches chart release naming convention for active sandbox & scenario names, then sidecar is injected
  - Otherwise it is ignored
- Chosen naming convention assumes sandbox name == sandbox namespace
- Sandbox name & namespace are currently strings with no format validation. Support for UUID-style formats have not been addressed in the scope of this feature release.

Reply:
OK - I assume that the hardcoded namespace will go away when UUID-style namespaces are supported
`40   const SandboxName = "sbox-1"`

Please enter a Jira issue to track this.

Reply:
Sandbox name is no longer hardcoded.